ui/app/routes/vault/cluster/policies/policy/edit.js

```diff
@@ -0,0 +1,3 @@
+import Route from '@ember/routing/route';
+
+export default class PolicyEditRoute extends Route {}
```

Comment:
It looks like the original route used the `unsaved-model-route` mixin so a similar approach could be taken to convert it to a base route class.